src/reducers/index.js

```diff
@@ -10,7 +11,8 @@ const reducers = {
     isLoggedIn,
     config,
     contacts,
+    contact,
```

Comment:
It might seem annoying, but would you mind alphabetizing these lists (imports and this list)? There's a great VSCode plugin that makes it super quick.